RecoJets/JetProducers/python/hltMVAJetPuId_cfi.py

```diff
@@ -0,0 +1,28 @@
+from RecoJets.JetProducers.hltPUIdAlgo_cff import *
+
+MVAJetPuIdCalculator = cms.EDProducer('MVAJetPuId',
```

Comment:
given how the name of the cfi file doesn't match the names of the module and there are two modules defined,
please 
- rename the file to hltMVAJetPuId_cff.py
- prepend hlt to the names of defined modules